core/binding/Cargo.toml

```diff
@@ -17,3 +17,6 @@ rocksdb = "0.12"
 lazy_static = "1.4"
 byteorder = "1.3"
 rlp = "0.4"
+futures = "0.3"
+async-trait = "0.1"
```

Comment:
Where is the `async-trait` used?

Reply:
used in tests/sdk.rs, to mock Storage trait

Reply:
move [devDependencies]